styles/styles.css

```diff
@@ -0,0 +1,398 @@
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 400;
+  src: url("fonts/ptsans-400.woff2") format("woff2");
```

Comment:
```suggestion
  src: url("../fonts/ptsans-400.woff2") format("woff2");
```

это какой урл? абсолютный или относительный?

Reply:
и начиная с этой домашки надо работу проверять в браузере - сразу бы увидела, что шрифты не приехали

Reply:
поправить везде